plugins/include/cstrike.inc

```diff
@@ -1231,6 +1231,84 @@ stock bool:cs_is_valid_itemid(id, bool:weapon_only = false)
 	return true;
 }
 
+/**
+ * Draws a HUD progress bar which is filled from 0% to 100% for the given
+ * amount of seconds. Once the bar is fully filled it will be removed from
+ * the screen automatically.
+ *
+ * @note If the "startpercent" argument is greater than 0, the bar will be
+ * filled from that amount of percentage instead of starting from 0%.
+ *
+ * @param id            Client index or 0 for all clients
+ * @param duration      How long (in seconds) until the bar is fully filled
+ * @param startpercent  Bar starting percentage (0-100)
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+native cs_draw_progress_bar(id, duration, startpercent = 0, bool:reliable = true)
```

Comment:
Missing semicolon.

Reply:
@WPMGPRoSToTeMa I'll run a "pragma semicolon 1" test tomorow and add them everywhere I forgot to.

Reply:
@WPMGPRoSToTeMa It's done now.